api/apps/api/src/modules/api-events/api-event.api.entity.ts

```diff
@@ -69,7 +69,7 @@ export class ApiEvent {
    */
   @ApiProperty()
   @IsEnum(Object.values(API_EVENT_KINDS))
-  @Column('enum')
+  @Column('enum', { enum: API_EVENT_KINDS })
```

Comment:
not true, it's `varchar`

Reply:
Could be that actual state isn't matched to what we have in entity declaration - still wanted a short route just o keep migration-generation to return results.

Reply:
then it's better to update it to `varchar` :D